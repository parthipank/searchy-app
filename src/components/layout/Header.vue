<template>
  <header class="header">
    <h1>** Searchy **</h1>
  </header>
</template>

<script>
export default {
  name: "Header"
};
</script>

<style scoped>
.header {
  background: teal;
  color: white;
  text-align: center;
  padding-top: 0px;
  padding: 10px;
}

.header a {
  color: whitesmoke;
  padding-right: 5px;
}
</style>