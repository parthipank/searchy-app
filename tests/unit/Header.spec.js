import { mount } from "@vue/test-utils";
import Header from "../../src/components/layout/Header";

it("Header Component", () => {
  const wrapper = mount(Header);
  expect(wrapper.element).toMatchSnapshot();
});
