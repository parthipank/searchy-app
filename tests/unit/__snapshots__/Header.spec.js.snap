// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component 1`] = `
<header
  class="header"
>
  <h1>
    ** Searchy **
  </h1>
</header>
`;
